internal/constants/constants.go

```diff
@@ -175,4 +175,4 @@ const (
 )
 
 // VersionInfo is the version of a binary. Left as a separate variable to allow override during build.
-var VersionInfo = "0.0.0"
+var VersionInfo = "v0.0.0"
```

Comment:
Our build process injects this value without the `v` prefix, so changing the default value to something else might have unexpected differences between unit tests and actual performance.
Also `vX.Y.Z` is [not valid semver](https://semver.org/#is-v123-a-semantic-version).

We should define a set way to label our binary builds, helm charts and container images to avoid confusion in the future.
I think currently we use `vX.Y.Z` for container images, and `X.Y.Z` for binary builds and helm charts.
Helm charts are a bit annoying since not all companies use the same scheme for version labeling. For example cert-manager uses `vX.Y.Z`.